File: examples/python/fanout/example_child_spawn.py
# > Child spawn
from examples.fanout.worker import ChildInput, child_wf

# 👀 example: run this inside of a parent task to spawn a child
child_wf.run(
    ChildInput(a="b"),
)

# > Error handling
try:
    child_wf.run(
        ChildInput(a="b"),
    )
except Exception as e:
    print(f"Child workflow failed: {e}")

File: sdks/ruby/examples/fanout/example_child_spawn.rb
# frozen_string_literal: true

require_relative "worker"

# > Child spawn
FANOUT_CHILD_WF.run({ "a" => "b" })
# !!

# > Error handling
begin
  FANOUT_CHILD_WF.run({ "a" => "b" })
rescue StandardError => e
  puts "Child workflow failed: #{e.message}"
end
# !!
